devito/passes/iet/parpragma.py

```diff
@@ -294,7 +295,8 @@ def _select_candidates(self, candidates):
                     except TypeError:
                         pass
 
-                collapsable.append(i)
+                if not i.is_ParallelAtomic or nested:
```

Comment:
Can you add a short explanatory comment?

Reply:
Actually we may want a careful explanation why we're peeking at is_ParallelAtomic here